api/storage/storage.proto

```diff
@@ -0,0 +1,34 @@
+syntax = "proto3";
+
+package sidero.discovery.storage;
+option go_package = "github.com/siderolabs/discovery-service/api/storagepb";
+
+import "google/protobuf/timestamp.proto";
+
+// StateSnapshot is the snapshot of the discovery service state.
```

Comment:
I'm torn on that, but feels like it _might_ make sense to move this down to https://github.com/siderolabs/discovery-api as a storage/snapshot proto.